ctapipe/reco/tests/test_hillas_intersection.py

```diff
@@ -332,3 +333,29 @@ def test_reconstruction_works(subarray_and_event_gamma_off_axis_500_gev):
     )
 
     assert reco_coord.separation(true_coord) < 0.1 * u.deg
+
+
+def test_selected_subarray(subarray_and_event_gamma_off_axis_500_gev):
```

Comment:
What does this test provide?
It looks like it does the same things as the `test_reconstruction_works` one

Reply:
It selects a subarray with telescopes 1, 4.

This tests fails without the fix, because of #1635

Reply:
Ah, sorry, I commited the changes to the original test by accident. Fixed now, good catch.